content/en/security/application_security/threats/security_signals.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Investigate Security Signals
+kind: documentation
+further_reading:
+  - link: "/security/default_rules/?category=cat-csm-threats#all"
+    tag: "Documentation"
+    text: "Explore CSM Threats detection rules"
+  - link: "/security/threats/workload_security_rules"
+    tag: "Documentation"
+    text: "Learn how to manage CSM Threats detection rules"
+  - link: "/security/notifications/"
+    tag: "Documentation"
+    text: "Learn more about security notifications"
+---
+
+## Overview
+
+ASM security signals are created when Datadog detects a threat based on a security rule. View, search, filter, and investigate security signals in the [Signals Explorer][2], or configure [Notification Rules][8] to send signals to third-party tools.
```

Comment:
> ASM security signals are created when Datadog detects a threat based on a security rule
Change security rule to detection rule. Detection Rules is the term used in the UI. [example](https://app.datadoghq.com/security/appsec/detection-rules?query=type%3Aapplication_security&deprecated=hide&groupBy=none&sort=rule_name):